[debug] [2022-12-16T10:18:14.338Z] ----------------------------------------------------------------------
[debug] [2022-12-16T10:18:14.340Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js emulators:start
[debug] [2022-12-16T10:18:14.340Z] CLI Version:   10.9.2
[debug] [2022-12-16T10:18:14.340Z] Platform:      win32
[debug] [2022-12-16T10:18:14.340Z] Node Version:  v16.16.0
[debug] [2022-12-16T10:18:14.342Z] Time:          Fri Dec 16 2022 19:18:14 GMT+0900 (대한민국 표준시)
[debug] [2022-12-16T10:18:14.343Z] ----------------------------------------------------------------------
[debug] 
[debug] [2022-12-16T10:18:14.346Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2022-12-16T10:18:14.431Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2022-12-16T10:18:14.431Z] > authorizing via signed-in user (slcksseo@gmail.com)
[debug] [2022-12-16T10:18:14.831Z] <<< [apiv2][status] GET https://firebase-public.firebaseio.com/cli.json 200
[debug] [2022-12-16T10:18:14.831Z] <<< [apiv2][body] GET https://firebase-public.firebaseio.com/cli.json {"cloudBuildErrorAfter":1594252800000,"cloudBuildWarnAfter":1590019200000,"defaultNode10After":1594252800000,"minVersion":"3.0.5","node8DeploysDisabledAfter":1613390400000,"node8RuntimeDisabledAfter":1615809600000,"node8WarnAfter":1600128000000}
[debug] [2022-12-16T10:18:14.908Z] java version "11.0.15.1" 2022-04-22 LTS

[debug] [2022-12-16T10:18:14.909Z] Java(TM) SE Runtime Environment 18.9 (build 11.0.15.1+2-LTS-10)
Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.15.1+2-LTS-10, mixed mode)

[debug] [2022-12-16T10:18:14.937Z] Parsed Java major version: 11
[info] i  emulators: Starting emulators: auth, database {"metadata":{"emulator":{"name":"hub"},"message":"Starting emulators: auth, database"}}
[warn] !  emulators: It seems that you are running multiple instances of the emulator suite for project yatchgame-ad28a. This may result in unexpected behavior. 
[debug] [2022-12-16T10:18:14.948Z] [hub] writing locator at C:\Users\aa\AppData\Local\Temp\hub-yatchgame-ad28a.json
[debug] [2022-12-16T10:18:15.261Z] > refreshing access token with scopes: []
[debug] [2022-12-16T10:18:15.263Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 <request body omitted>
[debug] [2022-12-16T10:18:15.523Z] <<< HTTP RESPONSE 200 {"pragma":"no-cache","cache-control":"no-cache, no-store, max-age=0, must-revalidate","date":"Fri, 16 Dec 2022 10:18:15 GMT","expires":"Mon, 01 Jan 1990 00:00:00 GMT","content-type":"application/json; charset=utf-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","server":"scaffolding on HTTPServer2","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3=\":443\"; ma=2592000,h3-29=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2022-12-16T10:18:15.533Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects/yatchgame-ad28a [none]
[debug] [2022-12-16T10:18:16.032Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects/yatchgame-ad28a 200
[debug] [2022-12-16T10:18:16.032Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects/yatchgame-ad28a {"projectId":"yatchgame-ad28a","projectNumber":"18941378065","displayName":"YatchGame","name":"projects/yatchgame-ad28a","resources":{"hostingSite":"yatchgame-ad28a","realtimeDatabaseInstance":"yatchgame-ad28a-default-rtdb"},"state":"ACTIVE","etag":"1_43e1c59f-5b61-4fdb-a0f3-a1ceb07f1ac6"}
[debug] [2022-12-16T10:18:16.033Z] database rules config:  [{"instance":"yatchgame-ad28a-default-rtdb","rules":"C:\\Users\\aa\\Downloads\\YatchGame-master\\database.rules.json"}]
[debug] [2022-12-16T10:18:16.038Z] Ignoring unsupported arg: projectId {"metadata":{"emulator":{"name":"database"},"message":"Ignoring unsupported arg: projectId"}}
[debug] [2022-12-16T10:18:16.038Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"database"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2022-12-16T10:18:16.038Z] Ignoring unsupported arg: rules {"metadata":{"emulator":{"name":"database"},"message":"Ignoring unsupported arg: rules"}}
[debug] [2022-12-16T10:18:16.038Z] Starting Database Emulator with command {"binary":"java","args":["-Duser.language=en","-jar","C:\\Users\\aa\\.cache\\firebase\\emulators\\firebase-database-emulator-v4.7.3.jar","--host","localhost","--port",9000],"optionalArgs":["port","host","functions_emulator_port","functions_emulator_host"],"joinArgs":false} {"metadata":{"emulator":{"name":"database"},"message":"Starting Database Emulator with command {\"binary\":\"java\",\"args\":[\"-Duser.language=en\",\"-jar\",\"C:\\\\Users\\\\aa\\\\.cache\\\\firebase\\\\emulators\\\\firebase-database-emulator-v4.7.3.jar\",\"--host\",\"localhost\",\"--port\",9000],\"optionalArgs\":[\"port\",\"host\",\"functions_emulator_port\",\"functions_emulator_host\"],\"joinArgs\":false}"}}
[info] i  database: Database Emulator logging to database-debug.log {"metadata":{"emulator":{"name":"database"},"message":"Database Emulator logging to \u001b[1mdatabase-debug.log\u001b[22m"}}
[debug] [2022-12-16T10:18:27.940Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"ui"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2022-12-16T10:18:27.940Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"ui"},"message":"Ignoring unsupported arg: port"}}
[debug] [2022-12-16T10:18:27.940Z] Starting Emulator UI with command {"binary":"node","args":["C:\\Users\\aa\\.cache\\firebase\\emulators\\ui-v1.7.0\\server.bundle.js"],"optionalArgs":[],"joinArgs":false} {"metadata":{"emulator":{"name":"ui"},"message":"Starting Emulator UI with command {\"binary\":\"node\",\"args\":[\"C:\\\\Users\\\\aa\\\\.cache\\\\firebase\\\\emulators\\\\ui-v1.7.0\\\\server.bundle.js\"],\"optionalArgs\":[],\"joinArgs\":false}"}}
[info] i  ui: Emulator UI logging to ui-debug.log {"metadata":{"emulator":{"name":"ui"},"message":"Emulator UI logging to \u001b[1mui-debug.log\u001b[22m"}}
[debug] [2022-12-16T10:18:28.113Z] Web / API server started at localhost:4000
 {"metadata":{"emulator":{"name":"ui"},"message":"Web / API server started at localhost:4000\n"}}
[debug] [2022-12-16T10:18:28.214Z] >>> HTTP REQUEST PUT http://localhost:9000/.settings/rules.json?ns=yatchgame-ad28a-default-rtdb  
 {
  "rules": {
    "player" : {
      "$uid" : {
        ".read" : "auth != null && auth.uid == $uid",
        ".write" : "auth != null && auth.uid == $uid"
      }
    },

    "lobby" : {
      ".read" : "auth != null && root.child('player').child(auth.uid).exists()",
      ".write" : "auth != null && root.child('player').child(auth.uid).exists()"
    },

    "waiting" : {
      "$roomId" : {
        ".read" : "auth != null",
        ".write" : "auth != null"
      }
    },

    "gameInfo" : {
      "$gameId" : {
        ".read" : "auth != null",
        ".write" : "auth != null"
      }
    },

    "rollDice" : {
      "$gameId" : {
        ".read" : "auth!=null",
        ".write" : "auth!=null"
      }
    },

    "boardInfo": {
      "$gameId" : {
        ".read" : "auth!=null",
        ".write" : "auth!=null"
      }
    }
  }
}
[debug] [2022-12-16T10:18:28.654Z] <<< HTTP RESPONSE 200 {"content-length":"15","content-type":"application/json; charset=utf-8","access-control-allow-origin":"*","cache-control":"no-cache","x-firebase-project-id":"yatchgame-ad28a-default-rtdb","x-firebase-project-number":"123456789","x-firebase-uuid":"db852edd-0685-46fc-8c0a-acad93fe99b7"}
[info] 
┌─────────────────────────────────────────────────────────────┐
│ ✔  All emulators ready! It is now safe to connect your app. │
│ i  View Emulator UI at http://localhost:4000                │
└─────────────────────────────────────────────────────────────┘

┌────────────────┬────────────────┬────────────────────────────────┐
│ Emulator       │ Host:Port      │ View in Emulator UI            │
├────────────────┼────────────────┼────────────────────────────────┤
│ Authentication │ localhost:9099 │ http://localhost:4000/auth     │
├────────────────┼────────────────┼────────────────────────────────┤
│ Database       │ localhost:9000 │ http://localhost:4000/database │
└────────────────┴────────────────┴────────────────────────────────┘
  Emulator Hub running at localhost:4400
  Other reserved ports: 4500

Issues? Report them at https://github.com/firebase/firebase-tools/issues and attach the *-debug.log files.
 
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: y3VGDsspt0jRQVM9TN3ZUIHmxQp0\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
[warn] !  Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf
    at AgentProjectState.getUserByLocalIdAssertingExists (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:208:19)
    at AgentProjectState.validateRefreshToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\state.js:295:24)
    at grantToken (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\operations.js:1212:38)
    at Proxy.<anonymous> (C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\lib\emulator\auth\server.js:375:20)
    at C:\Users\aa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\promise-breaker\index.js:265:46
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"metadata":{"emulator":{"name":"auth"},"message":"Error: Internal state invariant broken: no user with ID: Qz6odFOwnMTRxmVQMBuM05QiA6Mf\n    at AgentProjectState.getUserByLocalIdAssertingExists (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:208:19)\n    at AgentProjectState.validateRefreshToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\state.js:295:24)\n    at grantToken (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\operations.js:1212:38)\n    at Proxy.<anonymous> (C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\lib\\emulator\\auth\\server.js:375:20)\n    at C:\\Users\\aa\\AppData\\Roaming\\npm\\node_modules\\firebase-tools\\node_modules\\promise-breaker\\index.js:265:46\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
